### ISSUE TITLE
User can browse resources

### ISSUE THREAD

**Issue by benetechMatt** [benetech/ImageShare#6]
If a user doesn't have a specific thing they are looking for they will find it useful to "browse" by subject, resource type or accessibility accommodation.

On the main Home page, when the search field is left blank the "Subject", "Resource Type", and / or "Accessibility Accommodation" filters will be the only items which are part of the search criteria and all resources that match those filters will be displayed on the search results page.

**Comment by sinabahram** [comment]
I think this is fine, but @jkva  can you confirm once search is working that it behaves this way e.g. search term is optional if-only-if one or more filters are provided.

**Comment by jkva** [comment]
Implemented by 9c60d9a77d01dd5b00ca6af18fa3a11ab1faba64.

A search term is optional. If a search term is not provided and one or more filters are provided, these filters are applied (in an _AND_ relationship - subject AND type AND... for each set filter).

If a search term _is_ provided, this search term is applied in combination with any set filter in a most-relevant result set.
